packages/roosterjs-editor-dom/lib/index.ts

```diff
@@ -50,6 +50,7 @@ export { default as setColor } from './utils/setColor';
 export { default as matchesSelector } from './utils/matchesSelector';
 export { default as createElement, KnownCreateElementData } from './utils/createElement';
 export { default as moveChildNodes } from './utils/moveChildNodes';
+export { default as normalizeTableSelection } from './utils/normalizeTableSelection';
```

Comment:
If we can guarantee the selection is always normalized before assign to anywhere, do we still need to export this function?